src/monsters.cpp

```diff
@@ -1182,10 +1181,16 @@ void Monsters::loadLootContainer(const pugi::xml_node& node, LootBlock& lBlock)
 
 MonsterType* Monsters::getMonsterType(const std::string& name)
 {
-	auto it = monsters.find(asLowerCaseString(name));
+	std::string lowerCaseName = asLowerCaseString(name);
 
+	auto it = monsters.find(lowerCaseName);
 	if (it == monsters.end()) {
-		return nullptr;
+		auto it = unloadedMonsters.find(lowerCaseName);
```

Comment:
Change this var name or the other, VS complains about it.
`1>..\src\monsters.cpp(1188): warning C4456: declaration of 'it' hides previous local declaration`